packages/react-native-codegen/src/parsers/__tests__/error-utils-test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @oncall react_native
```

Comment:
Could you please enable flow in the test file? You can do it by adding this line in the license:

```suggestion
 * @flow strict-local
 * @oncall react_native
```